extensions/lifecycle/LifecycleConfigValidator.js

```diff
@@ -12,6 +12,7 @@ const joiSchema = joi.object({
     zookeeperPath: joi.string().required(),
     bucketTasksTopic: joi.string().required(),
     objectTasksTopic: joi.string().required(),
+    transitionTasksTopic: joi.string().required(),
```

Comment:
should it be required (i.e. require ZKOP to set the value on newer backbeat versions), or can we default to objectTasksTopic if this field is not specified (i.e. maintain backwards compatibility, degrading to the old behavior) ?